src/main/resources/view/ApplicationListCard.fxml

```diff
@@ -49,7 +49,7 @@
                        prefHeight="22"
                        minWidth="100"
                        alignment="CENTER"/>
-                <Label fx:id="status" styleClass="cell_small_label" text="\$status"/>
+                <Label fx:id="status" styleClass="cell_small_label" text="\$status" wrapText="true"/>
```

Comment:
Does status have wrapping issue? Iirc status only has 5 valid status and none of them requires wrapping (thus does this violate feature freeze?)